README.md

```diff
@@ -789,6 +789,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Srinivas](https://github.com/Mrdecent08)
   - [Sri Nivas](https://github.com/sri-nivas1227)
   - [Sri Venkat](https://github.com/srivenkat13)
+  - [Stephanie Kui](https://github.com/stephanie-kuihg)
```

Comment:
Please add your name in line 790, after Srinivas.

Reply:
Hey! @Panquesito7 current position is correct

Reply:
Hmm. According to a tool I used it's not correct. `r` comes first then `t` comes.

Reply:
Position is correct.

Alphabet goes `..pqRsTuvw...` -> r before t

Reply:
![image](https://user-images.githubusercontent.com/51391473/156190509-a5fd4d3c-4784-41ed-855e-d27c6ddf499b.png)

Reply:
Name with space often get this kind of  problem if we sort through a tool :)